openedx/features/enterprise_support/api.py

```diff
@@ -358,28 +361,63 @@ def get_enterprise_customer_cache_key(uuid, username=settings.ENTERPRISE_SERVICE
     )
 
 
-@enterprise_is_enabled()
```

Comment:
Removing the enterprise_is_enabled check, because it's innocuous to check the cache for a key, and presumably if any caller of this cares, that caller will do the check.